### PR TITLE
Upgrade various Python packages

### PR DESCRIPTION
**Pull request by ramen**
Tested on Win/Mac/Linux

Since the openai-whisper upgrade we did previously, the triton dependency is now OS-specific, which means we don't have to manually edit poetry.lock for Mac anymore!

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -38,14 +38,14 @@ torch = [
...
 ]
-jinja2 = "^3.1.2"
-celery = "^5.2.7"
-redis = "^4.5.4"
-ctranslate2 = "4.2.1"
-aiofiles = "^23.2.1"
+jinja2 = "^3.1.4"
+celery = "^5.4.0"
+redis = "^5.2.0"
+ctranslate2 = "4.4.0"
```
Comment (mikeylove) [inline pyproject.toml:44]: this is the one giving me problems now 🤔

looks like it's missing a mac x86_86 package 🤔🤔🤔

4.3.1 has x86_64 and nudging this down to that version fixes my run of `poetry install`

**Review by mikeylove — COMMENTED** [review]
pouring one out for intel macs ⚰️

**Review by mikeylove — APPROVED** [review]
it's working in this state so *fingers crossed*

**Comment by ramen** [comment]
> it's working in this state so _fingers crossed_

Thanks for your help!

**Pull request merged**